### ISSUE TITLE
Signature warning is shown, even when no visual signatures are present

### ISSUE THREAD

**Issue by jerbob92** [stephanrauh/ngx-extended-pdf-viewer#2635]
When opening a file that has a signature, but without a visible signature, it still shows the warning `This PDF file contains a digital signature. The PDF viewer can't verify if the signature is valid. Please download the file and open it in Acrobat Reader to verify the signature is valid.`

I think it's not needed to show the warning, since the user doesn't even see a signature.

When checking the PDF stream, I think an indicator of this is having a `/Rect` value of `[0, 0, 0, 0]` on the Annotation.

**Check this first**
The project is a wrapper around Mozilla's PDF viewer, so maybe your bug is caused by the base library. Please check this first by answering these three questions:
- Does the bug also happen on https://mozilla.github.io/pdf.js/web/viewer.html? (If so, please open the issue at Mozilla's bugtracker).

No, the warning comes from this library.

- Does the bug also happen on https://pdfviewer.net/simple?

Yes

- Can you write a reproducer based on a freshly created project? To create such a project, please use `ng new myissue` and `ng add ngx-extended-pdf-viewer`.

Yes, but probably not neccesary.

**Describe the bug**
A clear and concise description of what the bug is. For example:
1. Open a PDF with a signature, that's not visible.
2. The warning `This PDF file contains a digital signature. The PDF viewer can't verify if the signature is valid. Please download the file and open it in Acrobat Reader to verify the signature is valid.` is shown

**Version info**
- Version of ngx-extended-pdf-viewer

21.4.6

**Demo PDF file**
Many errors happen with specific PDF files only. So please add a PDF file (after checking the copyright of the PDF file first!)

https://www.tecxoft.com/samples/sample08.pdf

**Comment by stephanrauh** [comment]
Sounds like a good idea. At first, I didn't understand it - why should anybody add an invisible signature? But doing my research I learned that it does make sense in certain situations, and I'm pretty sure hiding the message doesn't create any security issue. If anybody disagrees, please tell me!

**Comment by jerbob92** [comment]
Awesome! Yeah what happens often is that organizations sign the PDF: "We made this PDF and verify by this invisible signature that we did and signed it at this date". 

This is different from a visible signature that's often put in there by a person "I signed this PDF (agree with the contents) and this digital signature is a proof of that it's actually my signature".

I think it's very good to show a warning with a visible PDF to show that it's not verified. For invisible PDF's nobody actually knows that it's their. Perhaps it would be good to show it _somewhere_, but probably not with a big fat red banner.
